pulp_rpm/app/serializers.py

```diff
@@ -911,7 +912,10 @@ class ModulemdSerializer(SingleArtifactContentUploadSerializer):
         help_text=_("Modulemd dependencies."),
         allow_null=True
     )
-    packages = serializers.PrimaryKeyRelatedField(
+    # TODO: The performance of this is not great, there's a noticable difference in response
+    # time before/after. Since this will only return Package content hrefs, we might benefit
+    # from creating a specialized version of this Field that can skip some of the work.
```

Comment:
The performance isn't *terrible* but it is noticable.  I timed it, requesting a page of modules takes about 2.6 seconds instead of 2.2 which is about 20% difference.